[Bug]: YaraAnalysisTaskTest test failure

### What steps will reproduce the bug?

Running github tests results in a test failure: https://github.com/google/turbinia/actions/runs/11843666733/job/33005112003#step:7:1173

### What is the expected behavior?

Tests should pass.

### What do you see instead?

```
======================================================================
FAIL: test_update_rules (workers.analysis.yara_test.YaraAnalysisTaskTest)
Tests the update_rules method
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/turbinia/turbinia/workers/analysis/yara_test.py", line 81, in test_update_rules
    self.assertEqual(ret, True)
AssertionError: False != True
```

Unsure of why this is failing now as it hasn't been touched in a while and tests passed in the latest PR affecting the relevant file and test: https://github.com/google/turbinia/commits/master/turbinia/workers/analysis/yara_test.py
https://github.com/google/turbinia/commits/master/turbinia/workers/analysis/yara.py


### Additional information

_No response_

--- a/turbinia/workers/analysis/yara.py
+++ b/turbinia/workers/analysis/yara.py
@@ -69,13 +69,16 @@ class YaraAnalysisTask(TurbiniaTask):
         repository = git.Repo(path)
         origin = repository.remotes.origin
         origin.pull(depth=1)
-        log.debug('Successfully updated rules from %s in %s', repo, path)
+        log.info('Successfully updated rules from %s in %s', repo, path)
       except git.exc.InvalidGitRepositoryError as e:
-        log.debug(
-            'InvalidGitRepositoryError updating rules in %s: %s', path, str(e))
+        log.error(
+            'InvalidGitRepositoryError updating rules in %s: %s', path, str(e),
+            exc_info=True)
         return False
       except Exception as e:
-        log.debug('Unknown error updating rules in %s: %s', path, str(e))
+        log.error(
+            'Unknown error updating rules in %s: %s', path, str(e),
+            exc_info=True)
         return False
 
     return True

